index.html

```diff
@@ -2363,68 +2370,131 @@ <h2 id="OfflineAudioCompletionEvent">
-                </div>
-              </li>
-              <li>For each dictionary member of the option dictionary passed
-              in, execute these steps, let <var>k</var> be its key, and
-              <var>v</var> be its value:
-                <ol>
-                  <li>If <var>k</var> is the name of an <a>AudioParam</a> on
-                  this <a>AudioNode</a> type, set the <a href=
-                  "#widl-AudioParam-value">value</a> attribute of this
-                  <a>AudioParam</a> to <var>v</var>.
-                  </li>
-                  <li>Else if <var>k</var> is the name of an attribute for this
-                  <a>AudioNode</a> type, set this attribute to <var>v</var>.
-                  </li>
-                  <li>Else if <var>k</var> is the name of a parameter to the
-                  associated <a>factory method</a>, set
-                  <var>parameters[i]</var> to <var>v</var>, where <var>i</var>
-                  is the index of the parameter in the signature for this
-                  <a>AudioNode</a> type.
-                  </li>
-                </ol>
-              </li>
-              <li>Call the corresponding <a>factory method</a> with the
-              parameters in <var>parameters</var>.
-              </li>
-            </ol>
-          </li>
-        </ol>
+        </p>
+        <section>
+          <h3>
+            AudioNode creation
+          </h3>
+          <p>
+            <a>AudioNode</a>s can be created in two ways: by using the
+            constructor for this particular interface, or by using the
+            <dfn>factory method</dfn> on the <a>BaseAudioContext</a> or
+            <a>AudioContext</a>.
+          </p>
+          <p>
+            The <a>BaseAudioContext</a> passed as first argument of the
+            constructor of an <a>AudioNode</a>s is called the <dfn id=
+            "associated">associated <a>BaseAudioContext</a></dfn> of the
+            <a>AudioNode</a> to be created. Similarly, when using the factory
+            method, the <a>associated <code>BaseAudioContext</code></a> of the
+            <a>AudioNode</a> is the first argument of the factory method.
+          </p>
+          <p>
+            To create a new <a>AudioNode</a> of a particular type <var>i</var>
+            using its constructor, with a <a>BaseAudioContext</a> <var>c</var>
+            as first argument, and an <a>associated option object</a>
+            <var>option</var> as second argument, from the <a href=
+            "https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global">
+            relevant global</a> of <var>c</var>, execute these steps:
+          </p>
+          <ol>
+            <li>Let <var>o</var> be a new object of type <var>i</var>.
+            </li>
+            <li>
+              <a href="#audionode-constructor-init">Initialize the common
+              part</a> of <var>o</var>, with <var>c</var> and <var>option</var>
```

Comment:
I assume you don't want to say "Initializing the common part" here anymore because that section isn't about just the common part anymore.